source/whats-new.txt

```diff
@@ -37,6 +38,29 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.6:
+
+What's New in 5.6
+-----------------
+
+The {+driver-short+} v5.6 release includes the following features:
+
+- The driver now supports Node.js v20.
+
+- The driver can return a cursor as the response to a server command by
+  calling the ``runCursorCommand()`` method. To learn more about this feature,
+  see the `runCursorCommand API documentation <{+api+}/5.6/classes/Db.html#command>`__.
```

Comment:
I: the link 404s

S: I think the /5.6/ isn't needed

Reply:
Strange, my last commit didn't seem to go through.